vNext/src/init-tools.cmd

```diff
@@ -26,6 +26,16 @@ if exist "%BUILD_TOOLS_SEMAPHORE%" (
   goto :EOF
 )
 
+for /f %%i in ('dotnet --version') do set INSTALLED_VERSION=%%i
+set /p REQUIRED_VERSION=<"%~dp0DotnetCLIVersion.txt"
+
+echo INSTALLED: %INSTALLED_VERSION%
+echo REQUIRED: %REQUIRED_VERSION%
+if '%INSTALLED_VERSION%' == '%REQUIRED_VERSION%' (
+  echo Will use system version of dotnet.
+  exit /b -1
+)
+
```

Comment:
Can you do this check outside this script? I'd like to avoid changes to this file as it's an exact copy/paste of https://github.com/dotnet/corefx/blob/master/init-tools.cmd and I'd like us to not fork and then have to maintain it separately from them. BTW, we should probably add a comment in the first line of this file saying that